components/dialogs/ReAuthDialog.vue

```diff
@@ -38,14 +38,14 @@ import {
 
 export default {
   name: 're-auth-dialog',
-  data() {
-    return {
-    };
-  },
   components: {
     BaseDialogV2,
     AuthCoreRegister,
   },
+  data() {
+    return {
+    };
+  },
   computed: {
```

Comment:
what for?

Reply:
Probably can remove, but lets make another PR...